src/ec/packages/ec-component-checkbox/__snapshots__/checkbox.test.js.snap

```diff
@@ -584,7 +584,7 @@ exports[`EC - checkbox Invalid renders correctly 1`] = `
       value="es"
     />
     <label
-      class="ecl-form-label ecl-checkbox__label ecl-form-label--invalid"
+      class="ecl-checkbox__label checkbox__label--invalid"
```

Comment:
checkbox__label--invalid seems strange? can't find correspondance in ecl

Reply:
indeed